deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "${aws_route53_record.origin.fqdn}"
+    origin_id   = "originRasterVisionSite"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
+    }
+  }
+
+  enabled          = true
+  http_version     = "http2"
+  comment          = "Raster Vision (${var.environment})"
+  retain_on_delete = true
+
+  price_class = "${var.cdn_price_class}"
+
+  # The trailing period at the end of the name is stripped off to comply with CloudFront's CNAME policy.
+  aliases = ["potsdam.${replace(data.aws_route53_zone.external.name, "/.$/", "")}"]
+
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originRasterVisionSite"
+
+    forwarded_values {
+      query_string = true
+      headers      = ["*"]
+
+      cookies {
+        forward = "none"
```

Comment:
In the tree prioritization demo we are passing all of the cookies through. I don't think this application makes use of cookies, but we should pass them through anyway because if we do at some point in the future, this may be hard to catch.